runbook.tf

```diff
@@ -0,0 +1,53 @@
+
+############ automation account  + runbook #############
+resource "azurerm_automation_runbook" "vm-start-stop" {
+  name                    = "${var.product}-vm-status-change-${var.env}"
+  location                = var.location
+  resource_group_name     = var.resource_group_name
+  automation_account_name = var.automation_account_name
+  log_verbose             = var.env == "prod" ? "false" : "true"
+  log_progress            = "false"
+  description             = "This is a powershell runbook used to stop and start ${var.product} VMs"
+  runbook_type            = "PowerShell"
+  content                 = file("${path.module}/vm-start-stop.ps1")
+
+  tags = var.tags
+}
+
+################# automation schedule #################
+resource "azurerm_automation_schedule" "vm-start-stop" {
+  for_each = { for aa_acc_runbook in var.auto_acc_runbooks : aa_acc_runbook.name => aa_acc_runbook }
```

Comment:
Can we change the var names to `var.schedules` ?

Reply:
Updated